Essentials/src/main/java/com/earth2me/essentials/commands/Commandessentials.java

```diff
@@ -253,7 +259,69 @@ private void runCleanup(final Server server, final CommandSource sender, final S
             }
             sender.sendMessage(tl("cleaned"));
         });
+    }
 
+    private void runHomes(final Server server, final CommandSource sender, final String commandLabel, final String[] args) throws Exception {
+        if (args.length < 2) {
+            sender.sendMessage("This sub-command provides a utility to mass-delete homes based on user options:");
+            sender.sendMessage("Use \"fix\" to delete all homes inside non-existent or unloaded worlds.");
+            sender.sendMessage("Use \"delete\" to delete all existing homes.");
+            sender.sendMessage("Use \"delete <worldname>\" to delete all homes inside a specific world.");
+            throw new Exception(HOMES_USAGE);
```

Comment:
```suggestion
            throw new Exception("/<command> homes (fix | delete [world])");
```
this is for consistency